git-gutter.el

```diff
@@ -227,14 +228,22 @@ gutter information of other windows."
     (with-temp-buffer
       (when (zerop (git-gutter:execute-command "git" t "rev-parse" "--is-inside-work-tree"))
         (goto-char (point-min))
-        (string= "true" (buffer-substring-no-properties
-                         (point) (line-end-position)))))))
+        (when (string= "true" (buffer-substring-no-properties
```

Comment:
Instead of returning a boolean, `git-gitter:in-git-repository-p` now returns either the root directory of the git repository, or `nil`. The `delete-backward-char` call removes the final newline of the `--show-toplevel` command's output.